.github/workflows/integration.yml

```diff
@@ -314,9 +314,10 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        k8s:
-          - v1.23
-          - v1.32
+        cases:
+        - {k8s: "v1.23", manage-controllers: false}
+        - {k8s: "v1.32", manage-controllers: false}
+        - {k8s: "v1.32", manage-controllers: true}
```

Comment:
How much time does adding a third case here add to our CI runs?

Reply:
None, these cases run in parallel. We could start experiencing resource contention the more parallel jobs we add, but I don't think I've seen that in this repo.

Reply:
We definitely see that when there are many concurrent jobs (dependabot, releases) running. It may be tolerable but it's another thing that will need retries/etc, so we shouldn't assume it's free.